shell/browser/ui/cocoa/atom_touch_bar.mm

```diff
@@ -368,16 +370,18 @@ - (void)updateButton:(NSCustomTouchBarItem*)item
   gfx::Image image;
   if (settings.Get("icon", &image)) {
     button.image = image.AsNSImage();
+  } else {
+    button.image = nil;
+  }
 
-    std::string iconPosition;
-    settings.Get("iconPosition", &iconPosition);
-    if (iconPosition == "left") {
-      button.imagePosition = NSImageLeft;
-    } else if (iconPosition == "right") {
-      button.imagePosition = NSImageRight;
-    } else {
-      button.imagePosition = NSImageOverlaps;
-    }
+  std::string iconPosition;
+  settings.Get("iconPosition", &iconPosition);
+  if (iconPosition == "left") {
+    button.imagePosition = NSImageLeft;
```

Comment:
Is it ok to set `imagePosition` when `image` is `nil`?

Reply:
I've tried it, it's just ignored when `icon` is `nil`.